test/Microsoft.AspNetCore.Authentication.Test/Google/GoogleMiddlewareTests.cs

```diff
@@ -781,6 +789,13 @@ private static TestServer CreateServer(GoogleOptions options, Func<HttpContext,
                         {
                             await context.Authentication.ChallengeAsync("Google");
                         }
+                        else if (req.Path == new PathString("/tokens"))
+                        {
+                            var authContext = new AuthenticateContext(TestExtensions.CookieAuthenticationScheme);
```

Comment:
What's the gun for?  I can't nuke the scheme because its a required argument, I could switch it to automatic scheme but that's just less clear I think